docker-hub/billing/downgrade.md

```diff
@@ -24,21 +24,23 @@ To downgrade from a Pro plan to a Free plan:
 
 1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The Billing page displays billing information about your existing user account.
+2. Open the drop-down menu next to your username in the top-right corner and select **Billing**. The **Billing Details** page displays billing information about your existing user account.
 
-3. Click **Change plan** to downgrade your existing plan.
+3. Ensure you are updating the billing details of your personal namespace.
```

Comment:
I'd be more explicit here, personally - something like

```suggestion
3. Ensure your personal namespace is selected in the list on the left of the page.
```

Reply:
Updated